lib/utils/device/deploy.ts

```diff
@@ -149,10 +149,11 @@ export async function deployToDevice(opts: DeviceDeployOptions): Promise<void> {
 		if (!semver.satisfies(version, '>=7.21.4')) {
 			exitWithExpectedError(versionError);
 		}
-		if (opts.live && !semver.satisfies(version, '>=9.7.0')) {
-			exitWithExpectedError(
-				new Error('Using livepush requires a supervisor >= v9.7.0'),
+		if (!opts.nolive && !semver.satisfies(version, '>=9.7.0')) {
+			globalLogger.logWarn(
+				`Using livepush requires a supervisor >= 9.7.0. A live session will not be started.`,
```

Comment:
Suggestion:
"Using livepush requires a **balena** supervisor **version** >= 9.7.0. A live session will not be started."

Reply:
Done